Link to Consent API

isharailanga opened the pull request:
- This PR is to address the gap in https://docs.google.com/document/d/1Qhtjwf1oakjBKo2DUbSVz8gpnQlZTxRfyiwS6mApLlI/edit
- Remove the WIP label
- Add the OB Accelerator Model diagrams

dimuthnc commented:
In the mentioned google doc, under the 2nd question, they have asked about how to retrieve consents by admin. This needs to be documented either in this[1] section (by providing a link) or this[2] section (with samples) in my opinion. Having the /admin call definition only in the reference section may not be helpful in my opinion.

[1] - https://ob.docs.wso2.com/en/latest/learn/consent-management/
[2] - https://ob.docs.wso2.com/en/latest/learn/consent-management-try-out/

isharailanga commented:
> In the mentioned google doc, under the 2nd question, they have asked about how to retrieve consents by admin. This needs to be documented either in this[1] section (by providing a link) or this[2] section (with samples) in my opinion. Having the /admin call definition only in the reference section may not be helpful in my opinion.
> 
> [1] - https://ob.docs.wso2.com/en/latest/learn/consent-management/ [2] - https://ob.docs.wso2.com/en/latest/learn/consent-management-try-out/

Addressed this concern by linking to the Consent REST API doc from `consent-management` page and adding a sample request to the `consent-manager` page.

<img width="719" alt="Screenshot 2022-03-04 at 00 25 23" src="https://user-images.githubusercontent.com/14251853/156633025-fdc1cd78-f052-4959-b75d-6704676ba6b6.png">


- Fixed for OB3.0 docs via https://github.com/wso2/docs-open-banking/pull/538/commits/0680b3a4ea65fa96e0aa8113af589ef6a1bbb59d.

- Fixed  for UK toolkit via bfc7bde3d6378707e1ab1c866ff50f5dc1b08cdd
